tests/test_app/test_app_mpi.py

```diff
@@ -4,6 +4,7 @@
 
 from cogent3.app import align as align_app
 from cogent3.app import io as io_app
+from cogent3.app import io_new as io_app_new
```

Comment:
delete the line with `io as io_app` and change `io_new ...` to` io_new as io_app`